src/openshift/openshiftItem.ts

```diff
@@ -32,10 +32,23 @@ export abstract class OpenShiftItem {
         return new Promise((res) => setTimeout(res, timeout));
     }
 
+    static openshiftData(data, value) {
```

Comment:
Why name a method that checks for name uniqueness 'openshiftData'? That makes zero sense.
Also, the type definitions are missing.